apis/Google.Bigquery.V2/Google.Bigquery.V2/BigqueryJob.cs

```diff
@@ -113,6 +115,48 @@ internal BigqueryJob(BigqueryClient client, JobList.JobsData resource)
         /// <returns>The final state of the job.</returns>
         public BigqueryJob Cancel(CancelJobOptions options = null) => _client.CancelJob(Reference, options);
 
+        /// <summary>
+        /// Asynchronously polls this job for completion.
```

Comment:
Not intentional, but not awful either. Adding them now for the sake of consistency...